client-react/src/App.css

```diff
@@ -1,17 +1,11 @@
 .App {
```

Comment:
Make this CSS file only scoped to App.tsx using module App.module.scss

Reply:
Done